src/lib/types/schedule.ts

```diff
@@ -33,6 +33,7 @@ export type SchedulePreset =
 export type ScheduleOffsetUnit = 'days' | 'hrs' | 'min' | 'sec';
 
 export type ScheduleActionParameters = {
+  identity?: string;
```

Comment:
This being allowable to undefined is potentially fucky I think? It's not immediately obvious to me why we wouldn't always supply identity